benches/README.md

```diff
@@ -4,7 +4,7 @@ The benchmarks are built with [criterion.rs](https://bheisler.github.io/criterio
 
 Measurements are taken on the elapsed wall-time.
 
-The benchmarks only focus on sucessfull transactions and vps: in case of failure, the bench function shall panic to avoid timing incomplete execution paths.
+The benchmarks only focus on sucessful transactions and vps: in case of failure, the bench function shall panic to avoid timing incomplete execution paths.
```

Comment:
```suggestion
The benchmarks only focus on successful transactions and vps: in case of failure, the bench function shall panic to avoid timing incomplete execution paths.
```